lib/reducers/__tests__/clock.spec.js

```diff
@@ -0,0 +1,36 @@
+import { expect } from 'chai';
+
+import { TYPES } from '../../actions/clock';
+import { clock } from '../clock';
+
+describe('clock reducer', () => {
+    it('should update the state with GET_TIME type', () => {
+        // Arrange
+        const state = {};
+        const action = {
+            type: TYPES.GET_TIME,
+            time: 'this is a test',
+        };
+
+        // Act
+        const result = clock(state, action);
```

Comment:
It might be a more representative test to call `clock(undefine, action)` and let the reducer handle the lack of a state through its initial state (RE testing best practice).